### PR TITLE
Add meta and assets packages

### PR DESCRIPTION
**Pull request by merelinguist**
# What are the changes and their implications?

Adds two new packages for internal consumption:

- `/meta`: default SEO/OG/etc tags for Blitz sites
- `/assets`: pre-built favicons and more for easily building branded pages

### Checklist

- [x] Tests added for changes
- [x] User facing changes documented

### Other information

Going to use this for the GUI package, can be used on any future Blitz sites (documentation, paid products, etc)

### CODE REVIEW

**Comment by merelinguist** [comment]
cc/ @flybayer?

**Comment by flybayer** [comment]
@merelinguist hey sorry, I took Sunday off work and yesterday/today I've been busy with client work and focusing on #233 in preparation for my livestream with Kitze tomorrow.

I'll try to review this tomorrow

**Comment by merelinguist** [comment]
Sorry no worries! Take your time :)

File: packages/meta/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "noFallthroughCasesInSwitch": true,
+    "moduleResolution": "node",
+    "baseUrl": "./",
+    "paths": {
+      "*": ["src/*", "node_modules/*"]
+    },
+    "jsx": "react",
+    "esModuleInterop": true
+  }
+}
```
Comment (flybayer) [inline packages/meta/tsconfig.json:23]: Ok finally reviewing this!

Is there a reason you didn't inherit from the top level tsconfig.json and then override here if needed?

**Comment by flybayer** [comment]
Is the assets package supposed to have a package.json?

File: packages/meta/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  color="#6700eb" // theme color
+  manifest="/site.webmanifest" // link to site manifest
+/>
+```
+
+All props are optional. If included multiple times, the tags from the last
+instance will be used.
+
+## Prior art:
+
```
Comment (lachlanjc) [inline packages/meta/README.md:33]: Would be great to say this code came from [my package](https://github.com/hackclub/theme/tree/master/packages/meta) :)

```suggestion
- https://github.com/hackclub/theme/tree/master/packages/meta
```

Comment (merelinguist) [reply packages/meta/README.md:33]: Sorry, thought I had included! Thx for the inspo. I think Blitz is just MIT globally, but I’m no expert on that.

File: packages/meta/tsconfig.json (hunk 1 above)
Comment (merelinguist) [reply packages/meta/tsconfig.json]: Yeah, meant to!

**Comment by merelinguist** [comment]
@flybayer Ready for review.

**Comment by flybayer** [comment]
@merelinguist this isn't hurting anything, but everything inside `packages/*` should have a package.json (and will get published to npm). If something doesn't need a package.json, then it should go somewhere else besides `packages/` :)

**Pull request merged**